ui/app/templates/components/auth-form.hbs

```diff
@@ -4,6 +4,31 @@
       {{partial 'svg/vault-loading'}}
     </div>
   {{/if}}
+  {{#if (has-feature "Namespaces")}}
+    <Toolbar>
+      <div class="field is-horizontal">
+        <div class="field-label is-normal">
+          <label class="is-label" for="namespace">Namespace</label>
+        </div>
+        <div class="field-body">
+          <div class="field">
+            <div class="control">
+              <input
+                value={{namespaceQueryParam}}
+                placeholder="/ (Root)"
+                oninput={{perform updateNamespace value="target.value"}}
+                autocomplete="off"
+                name="namespace"
+                id="namespace"
+                class="input"
+                type="text"
+              />
+            </div>
+          </div>
+        </div>
+      </div>
+    </Toolbar>
+  {{/if}}
```

Comment:
This will need some reworking in order to function properly, currently it errors because all of the action stuff lives on the route / controller and not this component. I think it might also look odd with the loading screen on top of the namespace input - previously it was outside of that so that you could still type while it fetches the new mounts. Is there any way to keep this where it was or is that not something we want to do?